cpp/src/io/parquet/page_data.cu

```diff
@@ -633,7 +634,7 @@ inline __device__ void gpuOutputString(volatile page_state_s* s,
  * @param[in] src_pos Source position
  * @param[in] dst Pointer to row output data
  */
-inline __device__ void gpuOutputBoolean(volatile page_state_s* s,
+inline __device__ void gpuOutputBoolean(volatile page_state_s*,
```

Comment:
This function is called only once thus probably you should remove that parameter completely.